src/js/edu-benefits/routes.jsx

```diff
@@ -1,126 +0,0 @@
-      getChildRoutes(partialNextState, callback) {
-        require.ensure([], (require) => {
-          callback(null, require('./1990e/routes').default);
-        }, 'edu-1990e');
-      },
-    },
-    {
-      path: '5490',
-      indexRoute: { onEnter: (nextState, replace) => replace('/5490/introduction') },
-      component: asyncLoader(() => {
-        return new Promise((resolve) => {
-          require.ensure([], (require) => {
-            store.replaceReducer(createCommonReducer(require('./5490/reducer').default));
-            resolve(require('./5490/Form5490App').default);
-          }, 'edu-5490');
-        });
-      }, 'Loading Form 22-5490'),
-      getChildRoutes(partialNextState, callback) {
-        require.ensure([], (require) => {
-          callback(null, require('./5490/routes').default);
-        }, 'edu-5490');
-      },
-    },
-    {
-      path: '1990n',
-      indexRoute: { onEnter: (nextState, replace) => replace('/1990n/introduction') },
-      component: asyncLoader(() => {
-        return new Promise((resolve) => {
-          require.ensure([], (require) => {
-            store.replaceReducer(createCommonReducer(require('./1990n/reducer').default));
-            resolve(require('./1990n/Form1990nApp').default);
-          }, 'edu-1990n');
-        });
-      }, 'Loading Form 22-1990N'),
-      getChildRoutes(partialNextState, callback) {
-        require.ensure([], (require) => {
-          callback(null, require('./1990n/routes').default);
-        }, 'edu-1990n');
-      },
-    },
-    {
-      path: '5495',
-      indexRoute: { onEnter: (nextState, replace) => replace('/5495/introduction') },
-      component: asyncLoader(() => {
-        return new Promise((resolve) => {
-          require.ensure([], (require) => {
-            store.replaceReducer(createCommonReducer(require('./5495/reducer').default));
-            resolve(require('./5495/Form5495App').default);
-          }, 'edu-5495');
-        });
-      }, 'Loading Form 22-5495'),
-      getChildRoutes(partialNextState, callback) {
-        require.ensure([], (require) => {
-          callback(null, require('./5495/routes').default);
-        }, 'edu-5495');
-      },
-    },
-    {
-      path: '1990',
-      indexRoute: { onEnter: (nextState, replace) => replace('/1990/introduction') },
-      component: asyncLoader(() => {
-        return new Promise((resolve) => {
-          require.ensure([], (require) => {
-            store.replaceReducer(createCommonReducer(require('./1990/reducer').default));
-            resolve(require('./1990/Form1990App').default);
-          }, 'edu-1990');
-        });
-      }, 'Loading Form 22-1990'),
-      getChildRoutes(partialNextState, callback) {
-        require.ensure([], (require) => {
-          callback(null, require('./1990/routes').default);
-        }, 'edu-1990');
-      }
-    }
-  ];
-
-  childRoutes.push({
-    path: '*',
-    onEnter: (nextState, replace) => replace('/')
-  });
```

Comment:
The Pensions app doesn't include this @jbalboni, is it correct to remove here?

Reply:
We should probably add it to pensions, actually.

Reply:
What about adding it in the `createRoutes` helper?